### DISCUSSION TITLE
@scalar/api-reference-react : Global CSS cannot be imported from within node_modules.

### DISCUSSION THREAD

**Discussion by gallirohik** [scalar/scalar#2351] [Q&A] [answered]
Hi Team, 
I'm following below example to integrate API docs on pages router. [alternative-pages-router](https://github.com/scalar/scalar/blob/main/documentation/nextjs.md#alternative-pages-router).
However, facing a weird problem, attaching screenshot for reference. 

FYI I'm using [nextra](https://nextra.site/docs).

Kindly help me with suitable fix.

thanks
Rohik
<img width="1720" alt="Screenshot 2024-07-02 at 3 29 36 PM" src="https://github.com/scalar/scalar/assets/28997354/31e85711-0c66-4331-9528-85211ef94654">

**Comment by hanspagel** [comment]
Oh no! I’ve created an issue here: https://github.com/scalar/scalar/issues/2352

**Reply by amritk** [reply]
We know what it is! I can fix right now

**Reply by gallirohik** [reply]
@amritk thank you. Pls let us know whenever patch version is live.
I tried to install updated package but it looks like its still in older version which is ```@scalar/api-reference-react@0.3.26``` instead of ```@scalar/api-reference-react@0.3.27 as mentioned in release notes```
<img width="853" alt="Screenshot 2024-07-03 at 1 07 34 PM" src="https://github.com/scalar/scalar/assets/28997354/9d52e88f-1f4e-41eb-a160-add480044e66">

Thanks
Rohik

**Reply by amritk** [reply]
Hey I checked and it looks like it is live. It might be npm's cache !

**Reply by gallirohik** [reply]
@amritk yes, its due to cache :( 

However, It seems we ran into similar issue with [RequestSidebarItem.vue2.js](https://github.com/scalar/scalar/blob/613def212cbccb9d0ffe4344f773835123c50e97/packages/api-client/src/views/Request/RequestSidebarItem.vue#L11) component as well. :( .


<img width="964" alt="Screenshot 2024-07-03 at 3 33 29 PM" src="https://github.com/scalar/scalar/assets/28997354/015b4d3a-b035-4ee6-86e2-739134f3b95b">

**Reply by amritk** [reply]
Oh I must have missed that one, (thought I got it). Will check right now!

**Reply by amritk** [reply]
Alright one more try https://github.com/scalar/scalar/pull/2375

**Reply by gallirohik** [reply]
Yeah! Thanks for ur prompt responses mate :) . Hopefully this fix will solve this bug :).

**Reply by amritk** [reply, marked as answer]
alright @gallirohik give `0.3.28` a try

**Reply by gallirohik** [reply]
@amritk & @hanspagel new version ```0.3.29``` working like a charm :). Thank you so much 
:)
